test_conformance/run_conformance.py

```diff
@@ -18,6 +18,9 @@
 import tempfile
 import string
 
+if sys.version_info[0] < 3:
+    import commands
```

Comment:
Why is `commands` being imported here for Python 2?  I cannot see any use of it.

Reply:
The original script imported commands, although it seems unnecessary. I just imported it to maintain what was like. I believe this can be removed.

Reply:
If it's not needed, please leave it removed.

Reply:
Now done.